recipes/arize-phoenix/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "arize-phoenix" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arize_phoenix-{{ version }}.tar.gz
+  sha256: 65a05713c8d727344878a3d1cc034ebb29642e598d91d9fc134c07481a8efddf
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.11
+    - hatchling
```

You need to explicitly depend upon `hatch-jupyter-builder` I think:
```suggestion
    - hatchling
    - hatch-jupyter-builder
```

---

Ah thank you so much @dopplershift